fontforge/http.c

```diff
@@ -663,7 +663,7 @@ return( true );
 FILE *URLToTempFile(char *url,void *_lock) {
     FILE *ret;
 
-    if ( strncasecmp(url,"http://",7)==0 )
+    if ( strncasecmp(url,"http://",7)==0 || strncasecmp(url,"https://",8)==0 )
```

Comment:
@frank-trampe I meant don't even have the ability to open URLs in FF. Just use whatever Python provides to download the file to some location, then pass that location to FF for it to load/process.